src/MasterController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Masterclass;
+
+use Aura\Di\Container;
+use FastRoute\Dispatcher;
+
+class MasterController
+{
+    private $config;
+
+    protected $container;
+
+    public function __construct(Container $container, $config)
+    {
+        $this->container = $container;
+        $this->config = $config;
+    }
+
+    public function execute()
+    {
+        $routeMap = $this->container->newInstance(RouteMap::class);
```

Comment:
Ah I see what you did here, Typically I'd advocate for injecting as much as you possibly can. We can't mock this, because I'm ALWAYS going to get an instance of RouteMap::class. I'd suggest injecting this in the __construct().